src/llama-model-loader.cpp

```diff
@@ -881,6 +885,18 @@ void llama_model_loader::load_data_for(struct ggml_tensor * cur) const {
     }
 }
 
+bool llama_model_loader::load_tensor(ggml_tensor * cur, const char * path, size_t file_offset, size_t tensor_offset, size_t size) {
+    if (!rpc_load_tensor_fn) {
+        return false;
+    }
+    ggml_backend_buffer_t buf = cur->view_src ? cur->view_src->buffer : cur->buffer;
+    const char * buf_name = ggml_backend_buffer_name(buf);
+    if (strncmp(buf_name, "RPC", 3) != 0) {
+        return false;
+    }
+    return rpc_load_tensor_fn(buf, cur, path, file_offset, tensor_offset, size);
+}
+
```

Comment:
@slaren Now that the signature of the function has been updated as per https://github.com/ggml-org/llama.cpp/pull/13106#issuecomment-2842578847, does it make sense for the function to become part of the backend interface? I think your idea was to make it like this so that the changes in `llama_model_loader` would become very minimal and there won't be special RPC-only logic and in the future to be able to implement it in other backends.